packages/dcos-history/extra/history/server_util.py

```diff
@@ -30,15 +41,6 @@ def headers_cb():
     defaults in this method. This method can be set by adding a dcos_auth_python package
     with a get_auth_headers method
     """
-    headers = {
-        "Access-Control-Allow-Credentials": "true",
-        "Access-Control-Allow-Headers": "accept, accept-charset, accept-encoding, " +
-                                        "accept-language, authorization, content-length, " +
-                                        "content-type, host, origin, proxy-connection, " +
-                                        "referer, user-agent, x-requested-with",
-        "Access-Control-Allow-Methods": "HEAD, GET, PUT, POST, PATCH, DELETE",
-        "Access-Control-Allow-Origin": "*",
-        "Access-Control-Max-Age": "86400"}
     if add_headers_cb:
         headers.update(add_headers_cb())
```

Comment:
This is now mutating a global value, so the additions to `headers` here are going to remain after `headers_cb()` returns, and potentially make their way into a response. You'll want to copy `headers` before you update it to prevent that from happening. Something like this:

```
return_headers = headers.copy()
if add_headers_cb:
    return_headers.update(add_headers_cb())
return return_headers
```

Reply:
🤦‍♂️ 

Thanks for catching this.